javascripts/main.js

```diff
@@ -0,0 +1,36 @@
+
+//**********//MAIN//**********//
+var petFood = function(object) {
```

Comment:
Right now, your IIFEs aren't _quite_ IIFEs.  They're missing the ```(``` before ```function``` and...